elide-core/src/main/java/com/yahoo/elide/core/PersistentResource.java

```diff
@@ -1364,10 +1366,10 @@ protected Object getValueChecked(String fieldName) {
      */
     protected Object getValueUnchecked(String fieldName) {
         requestScope.queueTriggers(this, CRUDAction.READ);
-        requestScope.queueTriggers(this, fieldName, CRUDAction.READ);
+        requestScope.queueTriggers(this, fieldName, CRUDAction.READ, Optional.empty());
         // Run the pre-security checks:
-        runTriggers(OnReadPreSecurity.class, "");
-        runTriggers(OnReadPreSecurity.class, fieldName);
+        runTriggers(OnReadPreSecurity.class, "", Optional.empty());
```

Comment:
Maybe we should extract this empty string into a constant like `CLASS_NO_FIELD = ""`?